autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>favor f-strings here and throughout:</p><p> </p><pre class="ql-syntax" spellcheck="false">base_job_name = f"{auto_ml_job_name}-{eval_obj}{timestamp_suffix}"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
done